lib/src/commands/test/test.dart

```diff
@@ -93,6 +94,12 @@ class TestCommand extends Command<int> {
             'should update the golden files.',
         negatable: false,
       )
+      ..addFlag(
+        'ansi-output',
+        defaultsTo: null,
+        help: 'Whether to force ansi output. If not specified, it depends on'
+            ' stdout and stderr',
```

Comment:
```suggestion
        help: 'Whether to force ansi output. If not specified, '
        'it will maintain the default behavior based on stdout and stderr.',
```

Reply:
@felangel  Thank you for review. Done.